README.md

```diff
@@ -85,7 +85,14 @@ the-job-that-runs-first:
 # ... Job 2 ...
 ```
 
-Then add the **new** **LAST JOB**
+**NOTE:**
+
+The step to start the workflow's trace should run first (before other jobs too). You do not need to start the trace in subsequent jobs.
+
+The output is important. This is the parent trace ID and will be used by the LAST job to connect all other traces.
```

Comment:
```suggestion
The output is important. This is the `trace-start` timestamp and will be used by the LAST job to ensure the duration of the full trace is correct.
```

Reply:
iirc, we don't require the trace id passed in at the end at this point because we can determine it reliably based on the inputs to the action (workflow name, timestamp, run attempt, etc) but the timestamp is important to get a nice waterfall.